src/lib/project.js

```diff
@@ -323,10 +340,67 @@ async function scaffoldNext() {
         ],
       },
     ];
-  }
+  },
 };`
   );
+  newNextConfig = newNextConfig.replace(
+    'reactStrictMode: true',
+    'reactStrictMode: false'
+  );
   fs.writeFileSync(path.join('ui', 'next.config.js'), newNextConfig);
+
+  const indexFileName = useTypescript === 'yes' ? 'index.tsx' : 'index.js';
+
+  fs.writeFileSync(
+    path.join('ui', 'pages', indexFileName),
+    newNextIndex,
+    'utf8'
+  );
+
+  // TODO: The command npm run build needs to be invoked in the smart contract directory root to generate
```

Comment:
comment can be removed, but would be good to add a comment to the build step to note why it's needed (it's obvious, but also potentially useful to future devs reading this code)